example.config.toml

```diff
@@ -1,28 +1,28 @@
 # Validator rpc end point(RPC of your own validator)
 #useful to gather information about network info, validatr voting power etc.
-validator_rpc_endpoint = "http://localhost:26657"
+validator_rpc_endpoint = "http://5.161.139.201:26657"
```

Comment:
in example.config should not give an exact endpoint. Just replace it with localhost.